debian/control

```diff
@@ -432,6 +462,18 @@ Description: Display server for Ubuntu - input platform library
  Contains the shared libraries required for the Mir server to interact with
  the input hardware using the evdev interface.
 
+Package: mir-platform-input-stub
+Section: libs
+Architecture: linux-any
+Multi-Arch: same
+Pre-Depends: ${misc:Pre-Depends}
+Depends: mir-platform-input-stub10
+Description: Display server for Ubuntu - stub input platform library
+ Mir is a display server running on linux systems, with a focus on efficiency,
+ robust operation and a well-defined driver model.
+ .
+ This package depends on nothing.
```

Comment:
Do we need this level of indirection? Can't `mir-test-tools` and `mirtest-dev` depend directly on the binaries?